Automattic/jetpack
# AI Assistant: Rename Breve and proofread entries on sidebar

Fixes #

## Proposed changes:

* Renames "AI Proofread" to "Write Brief with AI (BETA)". ref: pe4Cmx-2wX-p2#comment-1322
* Rename the proofread mentions to feedback
* Separate Feedback from Breve on the sidebar

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Check the Jetpack AI Assistant sidebar
* The text "AI Proofread" should not be displayed anymore
* The Feedback feature should be separated from Breve
* Breve and the Feedback feature should work as normal
* Add some text and go to the pre-publish sidebar
* Check that the Feedback feature works there as well
